packages/strapi-design-system/vite.config.ts

```diff
@@ -37,6 +38,6 @@ export default defineConfig(async () => {
         plugins: [typescript()],
```

Comment:
I'm not sure you need this plugin when `DTS === true` as the `dts` plugin will handle this instead.

Reply:
If I remove the typescript plugin with DTS true then I have this error when I try to build
<img width="1099" alt="typescript plugin" src="https://github.com/strapi/design-system/assets/2589748/80d088c0-32e8-48c2-b606-84993ef12321">